docs/en/changelog.md

```diff
@@ -1,6 +1,46 @@
 ## Changelog
 
 
+### V0.21 (1/29/2022)
+
+**Highlights**
+
+- Support Segmenter ([#955](https://github.com/open-mmlab/mmsegmentation/pull/955))
+- Support ISPRS Potsdam Dataset ([#1097](https://github.com/open-mmlab/mmsegmentation/pull/1097))
+- Support ISPRS Vaihingen Dataset ([#1171](https://github.com/open-mmlab/mmsegmentation/pull/1171))
+- Add UNet benchmark with multiple losses supervision ([#1143](https://github.com/open-mmlab/mmsegmentation/pull/1143))
+
+**New Features**
+
+- Support Segmenter ([#955](https://github.com/open-mmlab/mmsegmentation/pull/955))
```

Comment:
Add Segmenter info like:
Support GMA: Learning to Estimate Hidden Motions with Global Motion Aggregation (ICCV 2021)